demo/admin/src/products/ProductsGrid.tsx

```diff
@@ -54,6 +58,20 @@ function ProductsGridToolbar() {
             <ToolbarItem>
                 <GridColumnsButton />
             </ToolbarItem>
+            <ToolbarItem>
+                <CrudMoreActionsMenu
+                    overallActions={[
+                        {
+                            icon: <Excel sx={{ marginRight: "10px" }} />,
+                            label: <FormattedMessage {...messages.downloadAsExcel} />,
+                            disabled: exportApi.loading,
+                            onClick: () => {
+                                exportApi.exportGrid();
+                            },
+                        },
+                    ]}
```

Comment:
This is done differently than in #2573

Reply:
I think the admin-generator implementation is actually better/finished. so we probably don't need this pull request anymore. admin-generator should serve as demo and best-practice.

Reply:
I'd be okay with closing this PR, since we've already implemented it in the Admin Generator.